docs/development/pigs/pig-024.rst

```diff
@@ -71,30 +71,31 @@ Contributions to Gammapy can be made in different ways:
 - contributing to gammapy source code,
 - contributing to our documentation (rst pages, docstring, gammapy-webpage),
 - contributing to our DevOps environment (e.g. CI, docs and package deployment, GitHub configuration),
-- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, etc),
 - contributing to the maintenance of Gammapy products on open-source repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
 - leading the precise tasks identified by the Lead Developers (e.g module maintainer),
 - organizing and leading hands-on sessions or schools,
 - managing our social media accounts,
 - and coordinating the project by the Lead Developers.
 
-Any contribution without threshold on these topics renders a user as official Gammapy contributor. Many of these contributions
-are tracked using GitHub. However, the use of personal data is regulated in Europe and one should have the
-authorisation of users to be cited with their personal information (typically, their affiliation and their ORCID if any).
-For this reason, we propose to use the GitHub `Developer Certificate of Origin <https://github.com/apps/dco>`_ (DCO) on
-Pull Request. Its acceptation by contributors will permit to:
+Any contribution without threshold on these topics renders a user as official Gammapy contributor. Many of these
+contributions are tracked using the git history. However, the use of personal data is regulated in Europe and one
+should have the authorisation of users to be cited with their personal information for publications (their full name
+are mandatory, and if any their affiliation and their ORCID number). For this reason, we propose to use the
```

Comment:
optionally, we collect affiliations and ORCID numbers

Reply:
The use of 'if any' aims to promote this ORCID id. We aim to push for that as much as possible. In the case of the existence of the ORCID for a user, we gonna to use it.

Reply:
I also agree we should encourage the people to provide the ORCID if present. It is in their own interest and meta data will be more complete...

Reply:
This was mainly a language comment, it read a bit awkward. 

Maybe:

> (the full name is mandatory and we highly encourage also providing affiliation and ORCID)

?

Reply:
awkward? In which sense? how the wording 'if any' can be awkward?

Reply:
@bkhelifi I guess surrounding commas would make a difference as well. Like
```
are mandatory, and, if any, their affiliation and their ORCID number
```

Reply:
full name *is* mandatory